adafruit_motor/servo.py

```diff
@@ -41,10 +41,14 @@ class _BaseServo: # pylint: disable-msg=too-few-public-methods
        :param int min_pulse: The minimum pulse length of the servo in microseconds.
        :param int max_pulse: The maximum pulse length of the servo in microseconds."""
     def __init__(self, pwm_out, *, min_pulse=750, max_pulse=2250):
-        self._min_duty = int((min_pulse * pwm_out.frequency) / 1000000 * 0xffff)
-        max_duty = (max_pulse * pwm_out.frequency) / 1000000 * 0xffff
-        self._duty_range = int(max_duty - self._min_duty)
         self._pwm_out = pwm_out
+        self.set_pulse_widths(min_pulse, max_pulse)
+
+    def set_pulse_widths(self, min_pulse=750, max_pulse=2250):
```

Comment:
I like `set_pulse_width_range` more.

Reply:
I dropped the `s` too.

Reply:
Do you think it shouldn't have default values for the args, so you have to give both? I'm vacillating on this.

Reply:
Having default values was what motivated the original choice of name.

Reply:
I like defaults because it allows one to use the name in the call. aka `servo.set_pulse_width_range(min_pulse=700, max_pulse=2000)` You could even force it with: `def set_pulse_widths(self, *, min_pulse=750, max_pulse=2250):`